Description: fix non-zero exit status hardcoded in netstat.c-assorted_changes.patch
Bug-Ubuntu: https://bugs.launchpad.net/bugs/1251563
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=541172

--- a/netstat.c
+++ b/netstat.c
@@ -1761,12 +1761,14 @@
                  parsesnmp6(flag_raw, flag_tcp, flag_udp);
 #else
                  printf("Address type not supported for stats\n");
+                 exit(1);
 #endif
-            }
-            else
+            } else {
                  printf("Address type not supported for stats\n");
+		 exit(1);
+	    }
         }
-        exit(1);
+        exit(0);
     }
     
     if (flag_rou) {
